beacon-chain/rpc/validator_server.go

```diff
@@ -73,6 +75,23 @@ func (vs *ValidatorServer) WaitForActivation(req *pb.ValidatorActivationRequest,
 		}
 	}
 }
+// WaitTillSync checks if syncservice status is synched and sends message on stream when the service is synced.
+func (vs *ValidatorServer) WaitTillSync(_ *ptypes.Empty,stream pb.ValidatorService_WaitTillSyncServer) error{
```

Comment:
why does this need to be a stream ?

Reply:
it doesn't need to be a stream but we might later need this stream to know if we went out of sync and not just before initial sync

Reply:
this breaks on the client side , the moment we are synced. So there is no way to know if we went out of sync or not. I dont see why does this need to be a stream

Reply:
+1

Reply:
changed